src/components/product/Product.module.css

```diff
@@ -0,0 +1,94 @@
+.product__card {
+  padding: 20px 15px 15px;
+  border: 1px solid #CCC;
+  border-radius: 20px;
+  margin: 30px 58px 0;
```

Comment:
Please correct the gaps 
<img width="827" alt="image" src="https://github.com/demasha0908/fe-react-2024/assets/98031768/4b1b9f59-34d0-44c6-b8ba-217e4c41f975">

Reply:
If you use grid layout, the margins for its children are not needed. You can control inner indentation of the children by the grid parent `gap` prop.
Now your layout is broken 
<img width="1623" alt="image" src="https://github.com/demasha0908/fe-react-2024/assets/98031768/f548d457-bef6-40d2-b21f-054c6bcb1104">

Reply:
fixed